README.md

```diff
@@ -88,7 +88,7 @@ build+install the HIPCL library:
           ..
     make
 
-The samples directory contains some examples; these can be run from build directory, individually or via ctest.
+The samples directory contains quite a few examples; these can be run from build directory, individually or via ctest.
```

Comment:
I feel like this change contributes nothing; it also doesn't do what the commit message claims it does.

Reply:
Please pull whatever you feel right for you.